apps/modernization-ui/src/generated/models/EventId0.ts

```diff
@@ -0,0 +1,22 @@
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+export type EventId0 = {
```

Comment:
Is this an Investigation Event Id?

Reply:
yea, renamed one to `InvestigationEventId`, and the other to `LabReportEventId` for clarity